src/rulesets/SparseArrays/sparsematrix.jl

```diff
@@ -0,0 +1,15 @@
+using SparseArrays
```

Comment:
Not a full review, but just so it doesn't get missed, it is the pattern of this package, inline with BlueStyle,
to do all the `using`s at the top level file